### ISSUE TITLE
Missing features from STFMMLIB3D

### ISSUE THREAD

**Issue by hausler89** [flatironinstitute/FMM3D#43]
Hi,

I was wondering why this project seems to have "fewer" features than the old FMM3D project (from [here](https://cims.nyu.edu/cmcl/fmm3dlib/fmm3dlib.html)). My understanding was that this project is a continuation. Why was the half-space with no-slip boundary condition removed and should I just use the old code in that case?

Cheers!

**Comment by mrachh** [comment]
Yes, that would be the way to go for now. We are working on building those features back up in this repository.
